content/thesis/binary-fuzzing.md

```diff
@@ -3,7 +3,7 @@ title = "Binary Fuzzing Harness Generation"
 author = "Dominik Bayerl"
 keywords = ['Software Analysis', 'Binary Analysis']
 date = 2024-12-16
-expiryDate = 2025-04-01
+expiryDate = 2025-10-01

```

Comment:
This does not really fix the problem, it only delays it to october.
Please consider removing the expiryDate and/or adding a date conditional [here](https://github.com/securityinmobility/security.science/blob/main/content/career.md?plain=1#L25) to actually remove the entry when it has expired.